fossology/fossology
# feat(licenseRef): check flag before updating the license text

## Description

Do not update license text if it is already updated from UI.

### Changes

Add flag to license_ref table and check for different cases.

## How to test

* On a clean fossology instance change license text from ui and execute the postinstall scripts.
* Check if the updated license text still exist.
